app/controllers/songs_controller.rb

```diff
@@ -12,6 +12,12 @@ def create
     redirect_to setlist_path(id: params[:setlist_id])
   end
 
+  def destroy
+    Song.destroy(params[:id])
+    SetlistSong.destroy(params[:setlist_song_id])
```

Comment:
you are right, this was leftover from my song deletion attempts, not part of the setlist deletion feature.